eng/pipelines/templates/jobs/archetype-sdk-client.yml

```
@@ -6,249 +37,195 @@ jobs:
   condition: and(succeededOrFailed(), ne(variables['Skip.Test'], 'true'))
   strategy:
     matrix:
-      # Build with no dependencies at all (No samples) - PRECONDITIONS ON
-      # WARNINGS_AS_ERRORS          ON
-      # TRANSPORT_CURL              OFF
-      # UNIT_TESTING                OFF
-      # UNIT_TESTING_MOCKS          OFF
-      # TRANSPORT_PAHO              OFF
-      # PRECONDITIONS               ON
-      # LOGGING                     OFF
-      # AZ_PLATFORM_IMPL            AZ_PLATFORM_IMPL_NONE
-      Linux_ARM:
+      LinuxARM_Release_MapFiles_Logging:
         OSVmImage: 'ubuntu-18.04'
         vcpkg.deps: ''
+        VCPKG_DEFAULT_TRIPLET: 'x64-linux'
         AptDependencies: 'gcc-arm-none-eabi'
-        build.args: '-DCMAKE_TOOLCHAIN_FILE=cmake-modules/gcc-arm-toolchain.cmake'
+        build.args: '-DCMAKE_TOOLCHAIN_FILE=cmake-modules/gcc-arm-toolchain.cmake -DPRECONDITIONS=OFF'
         AZ_SDK_C_NO_SAMPLES: 'true'
+        PublishMapFiles: 'true'
+        MapFileArtifactSuffix: 'lnx-arm-rel-noprc-log'
+        BuildType: Release
 
-      # Build with no dependencies at all (No samples) - PRECONDITIONS OFF
-      # WARNINGS_AS_ERRORS          ON
-      # TRANSPORT_CURL              OFF
-      # UNIT_TESTING                OFF
-      # UNIT_TESTING_MOCKS          OFF
-      # TRANSPORT_PAHO              OFF
-      # PRECONDITIONS               OFF
-      # LOGGING                     OFF
-      # AZ_PLATFORM_IMPL            AZ_PLATFORM_IMPL_NONE
-      Linux_ARM_no_preconditions:
+      LinuxARM:
```

nit: For all the other columns, not specifying them in the name, to indicate "off", makes sense. However, we should still include `_Debug.`

```suggestion
      LinuxARM_Debug:
```